runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursorPlugin.java

```diff
@@ -44,7 +46,9 @@
 )
 public class CustomCursorPlugin extends Plugin
 {
-	private static final File customImageFile = new File(RuneLite.RUNELITE_DIR, "cursor.png");
+	private static final Logger logger = LoggerFactory.getLogger(CustomCursorPlugin.class);
```

Comment:
Annotate the class with `@Slf4j` and use the provided `log` object instead of this.